apm-agent-plugins/apm-es-restclient-plugin/apm-es-restclient-plugin-common/src/main/java/co/elastic/apm/agent/esrestclient/ElasticsearchRestClientInstrumentationHelper.java

```diff
@@ -104,8 +106,8 @@ public Span createClientSpan(String method, String endpoint, @Nullable HttpEntit
                 if (httpEntity != null && httpEntity.isRepeatable()) {
                     try {
                         IOUtils.readUtf8Stream(httpEntity.getContent(), span.getContext().getDb().withStatementBuffer());
-                    } catch (IOException e) {
-                        logger.error("Failed to read Elasticsearch client query from request body", e);
+                    } catch (Exception e) {
+                        queryExtractionErrorLogger.error("Failed to read Elasticsearch client query from request body", e);
```

Comment:
[minor] Keep the former exception handling and apply this log-once handling only for `UnsupportedOperationException`, where the message can say that this may be a know hibernatesearch issue.